files/repo/nodesource/NODESOURCE-GPG-SIGNING-KEY-EL

```diff
@@ -1,29 +1,30 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
why do we have a static key block for RPM, but Debian uses a remote URL? is it because there we can still pin the fingerprint of the key?